internal/internal_worker.go

```diff
@@ -962,7 +962,9 @@ func (aw *aggregatedWorker) Start() error {
 		}
```

Comment:
Why do we not skip worker start if there are not activities. And above when there are no workflows as well.

Reply:
In theory, it is possible to register workflow/activities after worker started.